api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -538,30 +570,6 @@ public void test_emailValidation_domain_ignoreCase() {
     assertThat(service.validateInstitutionalEmail(inst, "external-researcher@sanger.uk")).isFalse();
   }
 
-  @Test
```

Comment:
this become impossible now...

Reply:
I would like to keep this

Reply:
I can not prepare a state like this now because setting NO_ACCESS will not having that requirement in DB. So the test failed because of tier not found.

Reply:
let's drop this for now since we need a fix soon.  I'll make a note to come back to this.